eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgPaymentsDb.scala

```diff
@@ -247,19 +247,19 @@ class PgPaymentsDb(implicit ds: DataSource, lock: PgLock) extends PaymentsDb wit
   }
 
   private def parseIncomingPayment(rs: ResultSet): IncomingPayment = {
-    val paymentRequest = rs.getString("payment_request")
+    val invoice = rs.getString("payment_request")
     IncomingPayment(
-      PaymentRequest.read(paymentRequest),
+      Bolt11Invoice.fromString(invoice),
```

Comment:
Note: if you decide to change incoming payments to use the `Invoice` trait, you will need to use `Invoice.fromString` here (and in `SqlitePaymentsDb`).